ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -59,127 +49,113 @@ func NewSetupMixerAccountsPage(l *load.Load) *SetupMixerAccountsPage {
 // the page is displayed.
 // Part of the load.Page interface.
 func (pg *SetupMixerAccountsPage) OnNavigatedTo() {
-	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
```

Comment:
In many places there is an apparently unused ctx set. Is it ok to remove?

Reply:
yeah it is okay to remove i guess, if not used in on navigated from